### PR TITLE
Issue #401 - copied per-question statistics table from v1

### PR DESCRIPTION
**Pull request by parasgithub**

### CODE REVIEW

File: v2/pages/adminAssessment/adminAssessment.ejs [hunk 1]
```diff
@@ -245,6 +246,79 @@
...
       <div class="panel panel-primary">
         <div class="panel-heading">
+          <h3 class="panel-title"><%= assessment_set.abbrev %><%= assessment.number %> Stats</h3>
+        </div>
+
+        <table id="questionsTable" class="table table-condensed table-hover tablesorter">
+          <thead>
+            <tr>
+              <th>Question</th>
+              <th>Mean score</th>
```
Comment (mwest1066) [inline v2/pages/adminAssessment/adminAssessment.ejs]: Cell-center?

File: v2/pages/adminAssessment/adminAssessment.ejs [hunk 2]
```diff
@@ -245,6 +246,79 @@
...
+              <th class="cell-center">Discrimination</th>
+              <th class="cell-center">Attempts</th>
+              <th class="cell-center">Quintiles</th>
+            </tr>
+          </thead>
+          <tbody>
+            <% question_stats.forEach(function(row, i) { %>
+            <tr>
+              <td><%= row.number %> <%= row.title %></td>
```
Comment (mwest1066) [inline v2/pages/adminAssessment/adminAssessment.ejs]: Format like in the first panel on this page, with an extra period after the question number and the cell linked to the question page.

File: v2/pages/adminAssessment/adminAssessment.js [hunk 3]
```diff
@@ -202,8 +208,9 @@ router.post('/', function(req, res, next) {
         sqldb.queryOneRow(sql.select_finish_data, params, function(err, result) {
             if (ERR(err, next)) return;
             var credit = result.rows[0].credit;
-            var finish = true;
-            assessments.gradeAssessmentInstance(req.body.assessment_instance_id, res.locals.authn_user.id, credit, finish, function(err) {
+
```
Comment (mwest1066) [inline v2/pages/adminAssessment/adminAssessment.js]: Rebase error? There isn't an `assessmentExam.gradeExam()` function anymore, I don't think.

File: v2/pages/adminAssessment/adminAssessment.ejs (hunk 1 above)
Comment (parasgithub) [inline v2/pages/adminAssessment/adminAssessment.ejs]: Done in the next commit.

File: v2/pages/adminAssessment/adminAssessment.ejs (hunk 2 above)
Comment (parasgithub) [inline v2/pages/adminAssessment/adminAssessment.ejs]: Done in the next commit.

**Comment by parasgithub** [comment]
I think I accidentally deleted the "review" somehow, but I did make all the changes you requested.

**Comment by mwest1066** [comment]
Needs to be rebased onto current master so we can merge it.

**Pull request merged**